Makefile

```diff
@@ -39,15 +40,12 @@ lint:
 
 build: ## build cert-manager-istio-csr
 	mkdir -p $(BINDIR)
-	CGO_ENABLED=0 go build -o ./bin/cert-manager-istio-csr  ./cmd/.
+	CGO_ENABLED=0 go build -v -o ./bin/cert-manager-istio-csr  ./cmd/.
 
 verify: test build ## tests and builds cert-manager-istio-csr
 
-build_image_binary: ## builds image binary
-	GOARCH=$(ARCH) GOOS=linux CGO_ENABLED=0 go build -o ./bin/cert-manager-istio-csr-linux  ./cmd/.
-
-image: build_image_binary ## build docker image from binary
-	docker build -t quay.io/jetstack/cert-manager-istio-csr:v0.2.0 .
+image: ## build docker image targeting all supported platforms
+	docker buildx build --platform=$(IMAGE_PLATFORMS) -t quay.io/jetstack/cert-manager-istio-csr:v0.2.1 --output type=oci,dest=./bin/cert-manager-istio-csr-oci .
```

Comment:
I guess this make target won't work without setting up something like QEMU for multiarch? 
Would it perhaps make sense to create a Github action for the image building/pushing? (We'd just need to add Quay creds to repo secrets).
Alternatively may be worth to add some comments if any extra commands need to be run to set up Docker before running 'make image'?

Reply:
I would rather not use GitHub actions to push images since GA is notoriously bad for setting up and testing. I have been building the images locally and pushing them which I planned to keep doing.

I'll add a comment that if someone _is_ intending to push an image then they need to replace the `--output` to `--push`.

Reply:
Theoretically, I think the image building especially multi-arch 'just works' on GH actions - I've used it for two internal projects now and it's worked pretty much first time. Alternatively, there was a mention of using cmrel for building other cert-manager projects, I guess that would make the use of Quay creds less problematic. 
But in any case, any of that wouldn't be in this PR